notebooks/02_model/surprise_svd_deep_dive.ipynb

```diff
@@ -15,20 +15,21 @@
    "source": [
     "# Surprise Singular Value Decomposition (SVD)\n",
     "\n",
-    "\n",
-    "This notebook serves both as an introduction to the [Surprise](http://surpriselib.com/) library, and also introduces the 'SVD' algorithm which is very similar to ALS presented in the ALS deep dive notebook. This algorithm was heavily used during the Netflix Prize competition by the winning BellKor team."
+    "This notebook reviews the SVD algorithm in the [Surprise library](http://surpriselib.com/).  SVD is a widely used collaborative filtering method,  and part of [the winning solution](https://www.netflixprize.com/assets/GrandPrize2009_BPC_BellKor.pdf) in the Netflix Prize competition. "
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 1 Matrix factorization algorithm\n",
+    "## 1 Singular Value Decomposition\n",
+    "\n",
+    "SVD is a *matrix factorziation* technique, which we briefly review in the [ALS deep dive notebook](als_deep_dive.ipynb).  SVD is similar to ALS (Alternating Least Squares), with the following differences:\n",
     "\n",
-    "The SVD model algorithm is very similar to the ALS algorithm presented in the ALS deep dive notebook. The two differences between the two approaches are:\n",
+    "[comment]: <> (Should we add references below?)\n",
```

Comment:
I've been looking and in these notebooks: [NCF](https://github.com/Microsoft/Recommenders/blob/master/notebooks/02_model/ncf_deep_dive.ipynb), [VW](https://github.com/Microsoft/Recommenders/blob/master/notebooks/02_model/vowpal_wabbit_deep_dive.ipynb), [DKN](https://github.com/Microsoft/Recommenders/blob/master/notebooks/00_quick_start/dkn_synthetic.ipynb) and [xDeepFM](https://github.com/Microsoft/Recommenders/blob/master/notebooks/00_quick_start/xdeepfm_synthetic.ipynb) we have references at the bottom. I personally find them useful